test/uk/gov/hmrc/personaldetailsvalidation/connectors/FuturedValidationIdFetcherSpec.scala

```diff
@@ -30,12 +30,15 @@ import uk.gov.hmrc.personaldetailsvalidation.generators.ValuesGenerators._
 import uk.gov.hmrc.play.test.UnitSpec
 
 import scala.concurrent.ExecutionContext.Implicits.{global => executionContext}
+import scala.concurrent.duration._
 
 class FuturedValidationIdFetcherSpec
   extends UnitSpec
     with MockFactory
     with ScalaFutures {
 
+  override implicit val patienceConfig: PatienceConfig = PatienceConfig(timeout = 300 millis)
```

Comment:
This seems a little bit low. Jenkins slaves sometimes take more than this to execute futures. Would recommend making it a few seconds.